CommunityTracker/CommunityTracker.API/Controllers/CommunityController.cs

```diff
@@ -60,41 +61,61 @@ public async Task<IActionResult> GetAllManagers()
         /// <summary>
         /// Adds the community.
         /// </summary>
-        /// <param name="request">The API dto.</param>
+        /// <param name="apiDTO">The API dto.</param>
         /// <returns></returns>
         [HttpPost]
-        public async Task<IActionResult> AddCommunity([FromBody] AddRequestDTO request)
+        public async Task<IActionResult> AddCommunity([FromBody] AddRequestDTO apiDTO)
```

Comment:
```suggestion
        public async Task<IActionResult> AddCommunity([FromBody] AddRequestDTO apiDTO)
```

Revert apiDTO to request variable name